src/index.coffee

```diff
@@ -27,7 +27,8 @@ module.exports = (env, callback) ->
             url.push changefreq: page.metadata.changefreq
           if page.metadata.priority?
             url.push priority: page.metadata.priority
-        sitemap.urlset.push url: url
+        unless page.metadata?.noindex
```

Comment:
`getPages` is responsible for listing all pages that should be included in the sitemap.